lib/services/users.pm

```diff
@@ -161,7 +161,14 @@ sub full_users_check {
         change_pwd;
         add_user;
         #verify changed password work well in the following scenario:
-        lock_screen;
+        #workaround the lock screen will cause vnc lost connection issue on
+        #SLE15+ on s390x for bsc#1182958.
+        if ((check_var('ARCH', 's390x')) && is_sle('15+')) {
+            record_soft_failure("bsc#1182958, openQA test fails in install_service - gdm crashed after lock screen on s390x"); 
```

Comment:
Suggest also check migration scenario in if condition

Reply:
Now the test is only run in migration flavor, at least now we do not have to isolate this.